### PR TITLE
fix: lookup current shorting snx rewards rate & duration from contract

### PR DESCRIPTION
**Pull request by vbstreetz**
- Updates the weekly snx rewards to be looked up from the rewards contract.
- https://discord.com/channels/413890591840272394/549492529288970251/827537988304764948

### CODE REVIEW

File: sections/shorting/ShortingStats/ShortingStats.tsx [hunk 1]
```diff
@@ -16,14 +16,14 @@ import useSelectedPriceCurrency from 'hooks/useSelectedPriceCurrency';
 import { CRYPTO_CURRENCY_MAP } from 'constants/currency';
 import { NO_VALUE } from 'constants/placeholder';
 
-import { formatCurrency, formatPercent, toBigNumber, zeroBN } from 'utils/formatters/number';
+import { formatCurrency, formatPercent, zeroBN } from 'utils/formatters/number';
 import { WEEKS_IN_YEAR } from 'utils/formatters/date';
```
Comment (evgenyboxer) [inline sections/shorting/ShortingStats/ShortingStats.tsx:20]: seems like `WEEKS_IN_YEAR` is not used

**Comment by evgenyboxer** [comment]
@vbstreetz looks good! what has changed in the lock file?

File: sections/shorting/ShortingStats/ShortingStats.tsx (hunk 1 above)
Comment (vbstreetz) [reply sections/shorting/ShortingStats/ShortingStats.tsx]: Done 👍🏽

**Comment by vbstreetz** [comment]
> @vbstreetz looks good! what has changed in the lock file?

Not sure. Shall we gitignore lock files?

**Comment by vbstreetz** [comment]
Updated computation to https://github.com/Synthetixio/stats/blob/master/queries/yield-farming/useRewardsContractInfoQuery.ts#L79-L88

**Pull request merged**